docs/release/cut-sw-release.md

```diff
@@ -168,34 +182,30 @@ NON_UPCOMING_VERSIONS=<NON-UPCOMING VERSION(S)>
 
 dotest () {
     file=$dir/$client-$ver-1.$rhel.$arch.tar.gz
-    mkdir -p $rhel-$arch
-    pushd $rhel-$arch
-    tar xzf ../$file
-    $client/osg/osg-post-install
-    $client/osgrun osg-version
-    popd
-    rm -rf $rhel-$arch
+    if [ -e $file ]; then
+        echo "Testing $client-$ver-1.$rhel.$arch..."
+        mkdir -p $rhel-$arch
+        pushd $rhel-$arch
+        tar xzf ../$file
+        $client/osg/osg-post-install
+        $client/osgrun osg-version
+        popd
+        rm -rf $rhel-$arch
+    fi
 }
 
 pushd /tmp
 
-for client in osg-afs-client osg-wn-client; do
-    for ver in $NON_UPCOMING_VERSIONS; do
+for ver in $NON_UPCOMING_VERSIONS; do
+    for client in osg-afs-client osg-wn-client; do
```

Comment:
What's the reason for switching up the for loops?

Reply:
You don't know which client to test if you don't know the major version.